src/commands/workspace/upgrade.js

```diff
@@ -16,5 +18,11 @@ export function toWorkspaceUpgradeOptions(
 }
 
 export async function workspaceUpgrade(opts: WorkspaceUpgradeOptions) {
-  throw new BoltError('Unimplemented command "workspace upgrade"');
+  throw new BoltError(
+    `${chalk.red.bold(
+      '[bolt workspace upgrade]'
+    )} Unable to upgrade dependencies for a single workspace. \nIn order to upgrade a dependency [across all the workspaces] please run ${chalk.green.bold(
```

Comment:
Should we move these to messages.js?

Reply:
I observed logger messages are in messages js. But yeah I should be correct to move to messages js. I will do that.